lib/policies/load-balancer/balancer.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const httpProxy = require('http-proxy');
+const logger = require('../../logger').gateway;
+
+const ConfigurationError = require('../../errors').ConfigurationError;
+const RoundRobin = require('./round-robin');
+
+const strategies = {
+  'round-robin': RoundRobin
+};
+
+const createStrategy = (name, proxy, endpoints) => {
+  const Strategy = strategies[name];
+  return new Strategy(proxy, endpoints);
+};
+
+module.exports = function (params, config) {
+  const endpoints = params.serviceEndpoints
+    .map(name => {
+      const endpoint = config.gatewayConfig.serviceEndpoints[name];
```

Comment:
Another option would be that any config endpoint with `urls` instead of `url` gets round-robin treatment

```
serviceEndpoints:
  frontend:
    urls:
      - 'http://server1'
      - 'http://server2'
````